modules/oe_showcase_contact_forms/tests/src/ExistingSite/ContactFormTest.php

```diff
@@ -38,6 +48,14 @@ public function testCreateContactForm() {
     $this->markEntityTypeForCleanup('node');
     $this->markEntityTypeForCleanup('paragraph');
 
+    // Disable the flood limits for contact forms.
```

Comment:
There are two good reasons to add a comment:
1) the code is unclear, obfuscated and need clarifications;
here we add a comment explaining what the code does.
2) the code is clear but the reason for adding it is not;
here we add a comment why we added the code, the reason/motivation.

Here we are in the second situation, what the code does is clear, but the reason for adding it is not:
```suggestion
    // Disable the flood limits to avoid submit failure when run multiple times.
```